core/src/main/resources/lib/form/breadcrumb-config-outline.jelly

```diff
@@ -32,5 +32,5 @@ THE SOFTWARE.
   </st:documentation>
 
   <st:adjunct includes="lib.form.breadcrumb-config-outline.init"/>
-  <l:breadcrumb title="${attrs.title?:'%configuration'}" id="inpage-nav" />
+  <l:breadcrumb title="${attrs.title?:'%configuration'}" href="${rootURL}/configure" id="inpage-nav" />
```

Comment:
This makes no sense given this is a general purpose element.

Reply:
I can't quite remember to be honest! I've reverted this 👍